src/ruuvi.firmware.c.emProject

```diff
@@ -1009,8 +1255,8 @@
       linker_printf_fp_enabled="Float"
       linker_printf_width_precision_supported="Yes"
       linker_section_placement_file="$(ProjectDir)/targets/keijo/ses/flash_placement.xml"
-      linker_section_placement_macros="FLASH_PH_START=0x0;FLASH_PH_SIZE=0x30000;RAM_PH_START=0x20000000;RAM_PH_SIZE=0x6000;FLASH_START=0x26000;FLASH_SIZE=0xA000;RAM_START=0x20002d58;RAM_SIZE=0x32A8"
-      linker_section_placements_segments="FLASH RX 0x0 0x80000;RAM RWX 0x20000000 0x10000"
+      linker_section_placement_macros="FLASH_PH_START=0x0;FLASH_PH_SIZE=0x30000;RAM_PH_START=0x20000000;RAM_PH_SIZE=0x6000;FLASH_START=0x26000;FLASH_SIZE=0xA000;RAM_START=0x20002d58;RAM_SIZE=0x3D2A8"
```

Comment:
Why the memory size changes on unrelated board?

Reply:
Due to the large file size, I accidentally changed memory size. Fixed in latest commit.